src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1691,7 +1691,7 @@ private void CalculateIORedirection(bool isWindowsApplication, out bool redirect
 
             if (_runStandAlone)
             {
-                if (s_supportScreenScrape == null)
+                if (s_supportScreenScrape == null && Platform.IsWindows)
```

Comment:
Usually we use `#if !UNIX`.

Reply:
The suggestion was made by @SeeminglyScience to do it this way - I'll let the two of you sort out this conflicting feedback

Reply:
> The suggestion was made by @SeeminglyScience in conversation to do it this way - I'll let the two of you sort out this conflicting feedback

That was based on the assumption that `RuntimeInformation.IsOSPlatform` was a JIT intrinsic, which doesn't seem to be true anyway (even if it were used here). The ifdef is better 👍